src/Core/Resolvers/SqlMutationEngine.cs

```diff
@@ -156,6 +161,33 @@ public class SqlMutationEngine : IMutationEngine
                             result = GetDbOperationResultJsonDocument("success");
                         }
                     }
+                    // This code block contains logic for handling multiple create mutation operations.
+                    else if (mutationOperation is EntityActionOperation.Create && _runtimeConfigProvider.GetConfig().IsMultipleCreateOperationEnabled())
+                    {
+                        bool isPointMutation = IsPointMutation(context);
+
+                        List<IDictionary<string, object?>> primaryKeysOfCreatedItems = PerformMultipleCreateOperation(
+                                    entityName,
+                                    context,
+                                    parameters,
+                                    sqlMetadataProvider,
+                                    !isPointMutation);
+
+                        if (isPointMutation)
+                        {
+                            result = await queryEngine.ExecuteAsync(
+                                        context,
+                                        primaryKeysOfCreatedItems[0],
```

Comment:
need comment for why you take element `[0]` here.

Reply:
Added comment